app.py

```diff
@@ -158,7 +158,7 @@ def warning_color(self):
         :returns: An RGBA tuple of int (0-255).
         """
         color = sgtk.platform.qt.QtGui.QColor(self.style_constants["SG_ALERT_COLOR"])
-        return color.red(), color.green(), color.blue(), color.alpha()
```

Comment:
`SG_ALERT_COLOR` comes from tk-core and provides a hexadecimal color: `#FC6246` so no alpha.